17/umbraco-cms/customizing/extending-overview/extension-types/property-editor-schema.md

```diff
@@ -0,0 +1,216 @@
+| propertyEditorUiAlias    | string | Yes      | The Property Editor UI to use for editing this configuration value             |
+| config                   | object | No       | Optional configuration to pass to the Property Editor UI                       |
+| weight                   | number | No       | Optional ordering weight for the configuration field                           |
+
+### Settings Default Data Array
+
+Each object in the `defaultData` array provides default values:
+
+| Property | Type    | Required | Description                                   |
+| -------- | ------- | -------- | --------------------------------------------- |
+| alias    | string  | Yes      | The alias of the configuration property       |
+| value    | unknown | Yes      | The default value for this configuration      |
+
+## Complete Example
+
+```typescript
+import type { ManifestPropertyEditorSchema } from '@umbraco-cms/backoffice/property-editor';
+
+export const manifest: ManifestPropertyEditorSchema = {
+    type: 'propertyEditorSchema',
+    name: 'Suggestions Editor',
+    alias: 'My.PropertyEditor.Suggestions',
+    weight: 100,
+    meta: {
+        defaultPropertyEditorUiAlias: 'My.PropertyEditorUi.Suggestions',
+        settings: {
+            properties: [
+                {
+                    alias: 'maxChars',
+                    label: 'Maximum characters allowed',
+                    description: 'The maximum number of allowed characters in a suggestion',
+                    propertyEditorUiAlias: 'Umb.PropertyEditorUi.Integer',
+                    weight: 10,
+                },
+                {
+                    alias: 'placeholder',
+                    label: 'Placeholder text',
+                    description: 'Text displayed when the field is empty',
+                    propertyEditorUiAlias: 'Umb.PropertyEditorUi.TextBox',
+                    weight: 20,
+                },
+                {
+                    alias: 'disabled',
+                    label: 'Disabled',
+                    description: 'Disables the suggestion button',
+                    propertyEditorUiAlias: 'Umb.PropertyEditorUi.Toggle',
+                    weight: 30,
+                },
+            ],
+            defaultData: [
+                {
+                    alias: 'maxChars',
+                    value: 50,
+                },
+                {
+                    alias: 'placeholder',
+                    value: 'Write a suggestion',
+                },
+                {
+                    alias: 'disabled',
+                    value: true,
+                },
+            ],
+        },
+    },
+};
+```
+
+## Important Notes
+
+{% hint style="warning" %}
+The `alias` in the manifest **must exactly match** the alias used in the C# `DataEditor` attribute. The alias is the only connection between the server-side schema implementation and the client-side manifest.
+{% endhint %}
```

Comment:
How about we move this up to the "Manifest Properties" heading around line 74? That way you get this warning before you start working with the properties.